test/Groups/elements.jl

```diff
@@ -92,7 +92,7 @@ end
    @test eltype(PermGroup)==PermGroupElem
    @test eltype(PcGroup)==PcGroupElem
    @test eltype(FPGroup)==FPGroupElem
-   @test eltype(GL(2,3))==MatrixGroupElem{fqPolyRepFieldElem,fqPolyRepMatrix}
+   #@test eltype(GL(2,3))==MatrixGroupElem{fqPolyRepFieldElem,fqPolyRepMatrix}
```

Comment:
Why did this break?
It would be better to change to `@test_broken` instead of commenting if not possible to fix right now.
```suggestion
   @test_broken eltype(GL(2,3))==MatrixGroupElem{fqPolyRepFieldElem,fqPolyRepMatrix}
```

Reply:
```suggestion
   @test eltype(GL(2,3))==MatrixGroupElem{FqFieldElem, FqMatrix}
```
locally running shows this instead, which looks right

Reply:
I am afraid the test makes little sense and tests things which are not part of the API. These are exactly the tests, which make changing non-API things harder than they need to be.

Reply:
I will remove it.